src/components/App.js.jsx

```diff
@@ -3,8 +3,23 @@ import { Link } from 'react-router'
 import '../css/App.css'
 import BuilderContainer from './containers/BuilderContainer.js.jsx'
 import DashboardContainer from './containers/DashboardContainer.js.jsx'
+import Ajax from './helpers/ajax.js'
+import { connect } from 'react-redux' 
 
 class App extends Component {
+
+  componentDidMount() {
+    const api = new Ajax()
+    api.fetchTags().then(response => this.storeTags(response.data))
```

Comment:
@pwentz you're right, I totally forgot to mention that you need [redux-thunk](https://github.com/gaearon/redux-thunk). That middleware takes care of making sure your actions get `dispatch` and `getState`. Hopefully I didn't dump too many new concepts all at once :D, a lot of the stuff I suggested is nice but isn't all that essential to making your app actually work.